extensions/resteasy-reactive/quarkus-resteasy-reactive/deployment/src/main/java/io/quarkus/resteasy/reactive/server/deployment/QuarkusInvokerFactory.java

```diff
@@ -33,7 +37,52 @@ public QuarkusInvokerFactory(BuildProducer<GeneratedClassBuildItem> generatedCla
 
     @Override
     public Supplier<EndpointInvoker> create(ResourceMethod method, ClassInfo currentClassInfo, MethodInfo info) {
+        return info.parameters().stream().anyMatch(p -> p.name().toString().equals(CONTINUATION))
+                ? createCoroutineInvoker(method, currentClassInfo, info)
+                : createStandardInvoker(method, currentClassInfo, info);
+    }
+
+    private Supplier<EndpointInvoker> createCoroutineInvoker(ResourceMethod method, ClassInfo currentClassInfo,
```

Comment:
I would like to have some javadoc explaining what this actually generates

Reply:
it's actually the exact same as the standard invoker but it passes along the continuation argument as well.  I can add docs if you'd but that's the only real difference.